androidbrowserhelper/src/test/java/com/google/androidbrowserhelper/trusted/ManageDataLauncherActivityTest.java

```diff
@@ -0,0 +1,203 @@
+import static com.google.androidbrowserhelper.trusted.ManageDataLauncherActivity.CATEGORY_LAUNCH_SITE_SETTINGS;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.when;
+import static org.robolectric.Shadows.shadowOf;
+
+import static androidx.browser.customtabs.CustomTabsService.TRUSTED_WEB_ACTIVITY_CATEGORY;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.pm.ActivityInfo;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.content.pm.ServiceInfo;
+import android.net.Uri;
+import android.os.Build;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.annotation.Config;
+import org.robolectric.annotation.internal.DoNotInstrument;
+import org.robolectric.shadows.ShadowPackageManager;
+
+import androidx.browser.customtabs.CustomTabsService;
+import androidx.test.InstrumentationRegistry;
+
+/**
+ * Tests for {@link TwaProviderPicker}.
+ */
+@RunWith(RobolectricTestRunner.class)
+@DoNotInstrument
+@Config(manifest = Config.NONE)
+public class ManageDataLauncherActivityTest {
+    private PackageManager mPackageManager;
+    private ShadowPackageManager mShadowPackageManager;
+
+    private static final String TWA_PROVIDER_PACKAGE = "com.trustedweb.one";
+    private static final String CHROME = "com.android.chrome";
+    private static final int CHROME_72_VERSION = 362600000;
+    private static final int CHROME_71_VERSION = 357800000;
+
+    @Before
+    public void setUp() {
+        mPackageManager = RuntimeEnvironment.application.getPackageManager();
+        mShadowPackageManager = shadowOf(mPackageManager);
+    }
+
+    @Test
+    public void chrome72SupportsSiteSettings() {
+        installChrome(CHROME_72_VERSION);
+        TwaProviderPicker.Action action = TwaProviderPicker.pickProvider(mPackageManager);
+
+        assertTrue(ManageDataLauncherActivity.packageSupportsSiteSettings(action.provider, mPackageManager));
+    }
+
+    @Test
+    public void chrome71DoesNotSupportsSiteSettings() {
+        installChrome(CHROME_71_VERSION);
+        TwaProviderPicker.Action action = TwaProviderPicker.pickProvider(mPackageManager);
+
+        assertFalse(ManageDataLauncherActivity.packageSupportsSiteSettings(action.provider, mPackageManager));
+    }
+
+    @Test
+    public void browserWithCategorySupportsSiteSettings() {
+        installTrustedWebActivityProviderWithSiteSettingsCategory(TWA_PROVIDER_PACKAGE);
+
```

Comment:
nit: very small nit, but in the above two tests you don't have any newline between the "install..." and the "pickProvider" lines, whereas in these two tests you do.